VMEncryption/main/DiskUtil.py

```diff
@@ -1102,14 +1115,8 @@ def should_skip_for_inplace_encryption(self, device_item, encrypt_volume_type):
         if the answer is yes, then skip it.
         """
 
-        if encrypt_volume_type.lower() == 'data':
-            self.logger.log(msg="enabling encryption for data volumes", level=CommonVariables.WarningLevel)
-            if device_item.device_id.startswith('00000000-0000'):
-                self.logger.log(msg="skipping root disk", level=CommonVariables.WarningLevel)
-                return True
-            if device_item.device_id.startswith('00000000-0001'):
-                self.logger.log(msg="skipping resource disk", level=CommonVariables.WarningLevel)
-                return True
+        if encrypt_volume_type.lower() == 'data' and not self.is_data_disk(device_item, special_azure_devices_to_skip):
+            return True # Skip data disks
```

Comment:
A bit confused by the comment "Skip data disks".   I would have expected it to say "Skip non-data disk"

Reply:
sorry, yeah, it should be "non-data". Will fix.